codec-http2/src/main/java/io/netty/handler/codec/http2/internal/hpack/Encoder.java

```diff
@@ -351,10 +360,6 @@ private void add(byte[] name, byte[] value) {
             remove();
         }
 
-        // Copy name and value that modifications of original do not affect the dynamic table.
```

Comment:
@Scottmitch because we now pass in `CharSequence` which we expect to be immutable. Maybe I should add some javadocs to mention that ? That said its internal API anyway. WDYT ?